apps/ethereum_jsonrpc/config/config.exs

```diff
@@ -34,7 +34,8 @@ config :logger_json, :ethereum_jsonrpc,
 
 config :logger, :ethereum_jsonrpc, backends: [LoggerJSON]
 
-config :ethereum_jsonrpc, :internal_transaction_timeout, "900s"
+debug_trace_transaction_timeout = System.get_env("ETHEREUM_JSONRPC_DEBUG_TRACE_TRANSACTION_TIMEOUT", "900s")
```

Comment:
we should consider setting this in the helm charts - `debug_traceTransaction` timeouts are a big issue for us